swan-lake/learn-the-platform/test-document-the-code/generate-code-documentation.md

```diff
@@ -169,9 +169,9 @@ Now, let's add a function to the `math` module to be documented. Copy and paste
 
 ```ballerina
 # Calculates the value of the 'a' raised to the power of 'b'.
-# ```ballerina
-# float aPowerB = math:pow(3.2, 2.4);
-# ```
+# 
+float aPowerB = math:pow(3.2, 2.4);
+# 
```

Comment:
Incorrect? This is an example. Like https://github.com/ballerina-platform/ballerina-lang/blob/master/langlib/lang.array/src/main/ballerina/array.bal#L38

Reply:
We cannot have nested code snippets like https://github.com/ballerina-platform/ballerina-lang/blob/2beebd3126706e828fd3151f480a744b89a26a14/langlib/lang.array/src/main/ballerina/array.bal#L40 as they break the syntax highlighting 
![image](https://user-images.githubusercontent.com/7709843/227437888-04a89ae7-5362-4544-b286-48e87d75c3b0.png)

Reply:
Even when "```ballerina" is changed to use 4 backticks instead of 3? Either way, that needs to be fixed, right? This change is incorrect, but I don't think removing the example is correct either because it's required to actually show the feature here.